### DISCUSSION TITLE
DAkOmegaSST

### DISCUSSION THREAD

**Discussion by cczz8** [mdolab/dafoam#696] [General]
![屏幕截图 2024-10-14 164256](https://github.com/user-attachments/assets/434cd0af-f023-4618-bc5e-dac96f64e657)
Hello, Professor. I would like to understand why there is a Qsas term in the DAkOmegaSST turbulence model, while I have observed that there is no Qsas term in the OpenFOAM kOmegaSST turbulence model. Based on my knowledge, Qsas should be used in the kOmegaSSTSAS turbulence model.

**Comment by friedenhe** [comment]
It seems that this term is at least in the SST model in OpenFOAM-v1812: https://develop.openfoam.com/Development/OpenFOAM-plus/-/blob/OpenFOAM-v1812/src/TurbulenceModels/turbulenceModels/Base/kOmegaSST/kOmegaSSTBase.C#L199
